src/main/java/business/representation/RequestListRepresentation.java

```diff
@@ -69,6 +70,8 @@
 
     private Long dataAttachmentCount;
 
+    private ContactData billingAddress;
```

Comment:
Ik twijfel hier, want de `RequestListRepresentation` moet zo klein mogelijk blijven, zodat de hoeveelheid data voor het ophalen van alle aanvragen enigszins beperkt blijft.
Aan de andere kant gebruikt de `LabRequestRepresentation` (vanwege dezelfde reden) de `RequestListRepresentation` waar het benodige `billingAddress` niet in zit.
Wellicht is het in het geval van de pakbon handiger om de complete `RequestRepresentation` op te halen in de `LabRequestController`: https://github.com/thehyve/dntp-portal/blob/788e1869ff1df32d22c7c958fde716cd33299e76/src/main/resources/static/app/lab-request/LabRequestController.js#L201